src/js/game/GameController.js

```diff
@@ -219,13 +219,15 @@ class GameController {
         x: -450, alpha: 0.5
       }, this.timeout, Phaser.Easing.Linear.None);
 
+      tween.timeScale = 1;
       tween.start();
       tween = this.levelView.addResettableTween().to({
       }, this.timeout, Phaser.Easing.Linear.None);
 
       tween.onComplete.add(() => {
         this.endLevel(this.timeoutResult(this.levelModel));
       });
+      tween.timeScale = 1;
```

Comment:
I'd suggest a comment to that effect, since in isolation (especially in src rather than test) this seems odd and it's not clear why we're doing it.